skl2onnx/operator_converters/Flatten.py

```diff
@@ -0,0 +1,20 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ..proto import onnx_proto
+from ..common._apply_operation import apply_cast, apply_concat
```

Comment:
Why do you import these two when it is not being used?

Reply:
I did not see. I usually implement a unit test checking pep8 on my open source projects which I run before checking any change. So I forget to check that now. My mistake.